static/js/narrow.js

```diff
@@ -537,6 +537,7 @@ function pick_empty_narrow_banner() {
     } else if ((first_operator === "stream") && !stream_data.is_subscribed(first_operand)) {
         // You are narrowed to a stream to which you aren't subscribed.
         if (!stream_data.get_sub(narrow_state.stream())) {
+            //$(".compose_reply_button").hide();
```

Comment:
Probably should take out this comment if not needed.

Reply:
Ohh Sorry, I missed that